wpilibc/src/main/native/cpp/simulation/SendableChooserSim.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "frc/simulation/SendableChooserSim.h"
+
+using namespace frc::sim;
+
+SendableChooserSim::SendableChooserSim(std::string_view path) {
+  if constexpr (RobotBase::IsSimulation()) {
+    m_publisher = nt::NetworkTableInstance::GetDefault()
+                      .GetStringTopic(fmt::format("{}{}", path, "selected"))
+                      .Publish();
+  }
+}
```

Comment:
```suggestion
SendableChooserSim::SendableChooserSim(std::string_view path)
    : SendableChooserSim(nt::NetworkTableInstance::GetDefault(), path) {}

```
Matches Java, also adds missing blank line before the constructor right below. (Don't know if the indentation is right, though)